test/gtest/uct/test_uct_ep.cc

```diff
@@ -135,6 +135,52 @@ UCS_TEST_SKIP_COND_P(test_uct_ep, disconnect_after_send,
     }
 }
 
+UCS_TEST_SKIP_COND_P(test_uct_ep, is_connected,
+                     !has_ib() && !has_transport("tcp"))
+{
+    uct_ep_is_connected_params_t params;
+    uct_iface_attr_t iface_attr;
+    std::string dev_addr, ep_addr, iface_addr;
+    entity *e1, *e2;
+
+    create_sender();
+    connect();
+
+    e1 = create_entity(0);
+    e2 = create_entity(0);
+    e1->connect(0, *e2, 0);
```

Comment:
why need to create 3 entities? 2 should be enough.

Reply:
because the second entity is not created if "connect" is not called

Reply:
ok